backend/src/modules/typedefs.js

```diff
@@ -135,7 +135,9 @@ export const typeDefs = gql`
     changePassword(email: String!, oldPassword: String!, newPassword: String! newPasswordAgain: String!): Boolean!
     signin(email: String!, password: String!): AuthInfo!
     signup(
-      username: String!
+      username: String
```

Comment:
username už zas není povinný? :D

Reply:
přepoužívám signup mutation a organizace username nemá nebo má?

Reply:
áha, mě nedošlo, že to přepoužíváš, tak super

Reply:
původně jsem myslel, že to ani nepůjde, ale je to docela chytrý :)

Reply:
no jak na to koukam, tak nevim jestli je rozumne mit adresu jako jeden input field? spsi je potreba mit pro psc, street,city atd jednotlive fieldy
a upravit signup mutaci na backendu
a adresu nainsertuvat do tabulky place, nikoliv jako jeden sloupec pro organization

Reply:
done